delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/CompositeView.scala

```diff
@@ -26,7 +26,7 @@ final case class CompositeView(
     project: ProjectRef,
     sources: NonEmptySet[CompositeViewSource],
     projections: NonEmptySet[CompositeViewProjection],
-    rebuildStrategy: RebuildStrategy,
+    rebuildStrategy: Option[RebuildStrategy],
```

Comment:
would this work better if we were to have `RebuildStrategy.None`?

Reply:
Why?